src/NUnitFramework/tests/Attributes/TimeoutTests.cs

```diff
@@ -111,6 +111,36 @@ public void TimeoutCanBeSetOnTestFixture()
             Assert.That(result.ResultState, Is.EqualTo(ResultState.Failure));
             Assert.That(result.Message, Does.Contain("50ms"));
         }
+
+        [Test]
+        public void TestTimeOutNotElapsed()
```

Comment:
I like these new tests, but it would help to differentiate by name between simple tests and test cases, since one works and the other doesn't right now!